project/plugins.sbt

```diff
@@ -4,14 +4,14 @@ resolvers += "HMRC Releases" at "https://dl.bintray.com/hmrc/releases"
 
 resolvers += "Typesafe Releases" at "https://repo.typesafe.com/typesafe/releases/"
 
-addSbtPlugin("uk.gov.hmrc" % "sbt-auto-build" % "2.9.0")
+addSbtPlugin("uk.gov.hmrc" % "sbt-auto-build" % "2.13.0")
 
-addSbtPlugin("uk.gov.hmrc" % "sbt-git-versioning" % "2.1.0")
+addSbtPlugin("uk.gov.hmrc" % "sbt-git-versioning" % "2.2.0")
 
-addSbtPlugin("uk.gov.hmrc" % "sbt-play-cross-compilation" % "0.20.0")
+addSbtPlugin("uk.gov.hmrc" % "sbt-play-cross-compilation" % "1.2.0")
 
 val playPlugin = sys.env.getOrElse("PLAY_VERSION", "2.6") match {
```

Comment:
Should the default be 2.7 now?

Reply:
@matthewmascord Updated to use 2.7 as the default